### PR TITLE
more lenient linting

### PR DESCRIPTION
**Pull request by frodopwns**
**What this PR does / why we need it**:
- Increase max line length
- remove forced alpha ordering of things
- allow console.log (for now)
- allow _ prefixed var names

### CODE REVIEW

**Comment by ericpera-msft** [comment]
@frodopwns Do we want to enforce single quotes vs. double quotes? When I enforce the new tslint file, I get a ton of linter errors about double quotes in all my files.

File: tslint.json [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-}
\ No newline at end of file
+      "max-line-length": {
+        "options": [150]
+      },
+      "member-ordering": false,
+      "no-consecutive-blank-lines": true,
+      "object-literal-sort-keys": false,
+      "ordered-imports": false,
+      "quotemark": [true, "single"],
```
Comment (ericpera-msft) [inline tslint.json]: @frodopwns Do we want to enforce single quotes? When I enforce the new tslint file, I get a ton of linter errors about double quotes in all my files.

Comment (frodopwns) [reply tslint.json]: We were already enforcing them weren't we?

Comment (frodopwns) [reply tslint.json]: interesting, guess not

Comment (ericpera-msft) [reply tslint.json]: Whether or not you want to change that is up to you, but everything else looks good otherwise.

Comment (frodopwns) [reply tslint.json]: Looks like we are enforcing doubles currently

Comment (ericpera-msft) [reply tslint.json]: Do we want to keep that?

Comment (frodopwns) [reply tslint.json]: we should enforce using one for consistency, don't personally care which one

**Pull request merged**